src/components/Payment/component/SuccessPageButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "styled-components";
+
+const SuccessPageButton = ({ isMatched, postButtonOn, setPostButtonOn }) => {
```

Comment:
```suggestion
const SuccessPageButton = ({ isMatched, setPostButtonOn }) => {
```
아래 함수를 함수형 업데이트로 하게 되면 Props로 state를 넘겨주지 않을 수 있습니다!